configBase.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const axios = require('axios')
+const FormData = require('form-data')
+const fs = require('fs')
+const path = require('path')
+
+const SERVICE_NAME = process.env.SERVICE_NAME
+const API_PASSWORD = process.env.API_PASSWORD
+const SERVICE_API_PORT = process.env.SERVICE_API_PORT
+const API_PATH = process.env.API_PATH
+const API_USERNAME = process.env.API_USERNAME
+const SSL = process.env.SSL
+const BODY_DATA_FORMAT = process.env.BODY_DATA_FORMAT
```

Comment:
I think MIME_TYPE might be a more fitting name for this

Reply:
For sure, changed in [3dc8673](https://github.com/jembi/api-config-importer/pull/1/commits/3dc8673f60f1260e1d11e8e07269492fa99a8494)